library/modules/EventManager.cpp

```diff
@@ -697,6 +700,27 @@ static void manageSyndromeEvent(color_ostream& out) {
     lastSyndromeTime = highestTime;
 }
 
+static void manageUnitStressEvent(color_ostream& out) {
```

Comment:
I don't think running too often is a concern. It's pretty similar to manageUnitDeathEvent().

To stop it triggering all the time for the same unit it could trigger on a STRESSED_CITIZEN announcement. Is it safe to use announcements as signals or can it get screwed up by ingame announcement settings?

Reply:
Yeah, the current check would have it triggering every time it runs while a unit has stress >= 250000. I'll look into using the announcement.